engine/src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -391,61 +399,43 @@ class SlickDataAccess(databaseConfig: Config) extends DataAccess {
     runTransaction(action)
   }
 
+  private def futureDescriptorFromExecutionAndAux(executionAndAux: ExecutionAndAux)(implicit ec: ExecutionContext): Future[WorkflowDescriptor] = {
```

Comment:
not that I *really* care though most methods in this file return futures but don't have 'future' in the name of the method... maybe `workflowDescriptorFromExecutionAndAux`?